src/api/models/v2/lesson.model.ts

```diff
@@ -0,0 +1,58 @@
+import { DataTypes, Model, Optional } from 'sequelize';
+import { sequelize } from '../../../config/databasev2';
+import { Module } from './module.model';
+
+export interface LessonAttributes {
+  id: string;
+  title: string;
+  content: string;
+  hidden: boolean;
+  moduleId?: string;
+  orderNumber: number;
```

Comment:
Leave a comment here on what is orderNumber: the occurrence of lesson in the module.